atd-vze/src/views/Locations/Location.js

```diff
@@ -16,8 +32,94 @@ function Location(props) {
   if (loading) return "Loading...";
   if (error) return `Error! ${error.message}`;
 
+  const getAggregatePersonsSum = (data, field) => {
+    return (
+      data.atd_txdot_primaryperson_aggregate.aggregate.sum[field] +
+      data.atd_txdot_person_aggregate.aggregate.sum[field]
+    );
+  };
+
+  const vehBodyGraphConfig = {
+    palette: palette(
```

Comment:
This google-palette seems cool but is new to me. What does it do here? Will it keep our colors consistent with the colors defined from CoreUI that Mike inventoried in `src/styles/colors.js`?

_looks at page with example chart_

Now seeing at the page you recommended, I see what is happening and the colors do kinda conflict with our theme. Tempted to go in the weeds here and start saying we should group some of these vehicle types and use the same colors for likes, but I'm just gonna say this is a great first pass, let's leave it as is and restyle once we understand if its useful.

Would like to see a precent more than an absolute number in the donut graph though.

Reply:
This will have to be another branch, the graph was failing due to the lack of colors defined in an array. I figured using palette to generate those colors for me in the mean time, there are many templates (palettes) we can pick from, or we can hand-pick our own colors in that new branch.